src/tests/ftest/mdtest/mdtest_small.py

```diff
@@ -15,6 +16,7 @@ class MdtestSmall(MdtestBase):
     :avocado: recursive
     """
 
+    @skipForTicket("DAOS-7304")
```

Comment:
I think this issue might be duplicate of DAOS-6798 and should be resolved on master

Reply:
Thats good, it raises the question on why it wasn't skipped before?

Reply:
If you follow the comments in the ticket, the issue was resolved at some point but resurfaced and reopened. During the time the ticket is 'open', you'll see a low occurrence of the issue, which is why we did not disable the test, similar reason to daoscoretest checksum.